package.json

```diff
@@ -16,7 +16,7 @@
     "format": "prettier --write \"{src,test}/**/*.js\"",
     "lint": "eslint src/**",
     "test:debug": "node --lazy --inspect-brk test",
-    "test": "node test && git diff -- test/*.Formatted.cs",
+    "test": "node test && git diff --exit-code -- test/*.Formatted.cs",
```

Comment:
From what I understand, this option will just hide the output of the diff. Exit code is the same, so I'm not sure what this is supposed to fix. Can you detail why we need this?